libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/mysql/MySQLNoLessThan5700SchemaAccessor.java

```diff
@@ -1350,14 +1350,22 @@ public DBProcedure getProcedure(String schemaName, String procedureName) {
                     .identifier(rs.getString("PARAMETER_NAME")).space()
                     .append(rs.getString("DTD_IDENTIFIER")).append(",");
         });
+        MySQLSqlBuilder getDDL = new MySQLSqlBuilder();
```

Comment:
1. `parameters` is never used, the `queryForParameters` is unnecessary.
2. please add more unit tests for `getProcedure` to ensure the parser can handle the show create DDL.

Reply:
ok